src/ui/molecules/bridge.transfer/bridge.transfer.fee.js

```diff
@@ -23,19 +23,34 @@ export const BridgeTransferFee = ({
   }, [amount, token]);
   // Getting Fees - END
 
+  useEffect(() => {
+    if (fee && amount) {
+      setFee(null);
```

Comment:
Why are we setting fee to null here?  Seems like the useEffect above should be handling this.

Reply:
Added comments and cleaned up code to make this more clear. The issue is that if you go from ETH to an wrapped BTC asset it will error because parseUnits can't handle 18 decimal places.